ci/cudf_pandas_scripts/pandas-tests/diff.sh

```diff
@@ -8,17 +8,22 @@
 
 # Hard-coded needs to match the version deduced by rapids-upload-artifacts-dir
 MAIN_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py310.main-results.json
-PR_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py310.pr-results.json
-aws s3 cp $MAIN_ARTIFACT main-results.json
+PR_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py39.pr-results.json
```

Comment:
The python version changed and is different from the main artifact. Is that expected?

Also, why not use Python 3.11?

Reply:
answered here: https://github.com/rapidsai/cudf/pull/15485#discussion_r1556345903

There seems to be not python 3.11 for cuda-12.2 `pull-request` build type.